tools/releaseBuild/azureDevOps/templates/nuget.yml

```
@@ -10,7 +10,7 @@ jobs:
   pool:
     name: PowerShell1ES
     demands:
-    - ImageOverride -equals MMS2019TLS
+    - ImageOverride -equals MMS2019
```

```suggestion
    - ImageOverride -equals MMS2019TLS
```

We shouldn't revert this change